package.json

```diff
@@ -39,7 +40,7 @@
     "ember-cli-ic-ajax": "0.2.4",
     "ember-cli-inject-live-reload": "^1.4.0",
     "ember-cli-jshint": "^1.0.0",
-    "ember-cli-mirage": "0.2.0",
+    "ember-cli-mirage": "^0.2.0",
```

Comment:
If we want to merge this, we will have to link to my PR for now. Otherwise, tests will fail.